querybook/server/lib/event_logger/base_event_logger.py

```diff
@@ -12,6 +15,60 @@ def logger_name(self) -> str:
         """Name of the event logger that will be shown on the frontend"""
         raise NotImplementedError()
 
+    @property
+    def api_allow_list(self) -> list:
+        """API endpoints from this list will be logged.
+        If None is returned, all endpoints will be loggeed.
+
+        You can override this property to provide your own list in your logger.
+        """
+        return None
+
+    @property
+    def api_block_list(self) -> list:
+        """API endpoints from this list will not be logged. This list will only be used
+        when the allow list is None.
+
+        You can override this property to provide your own list in your logger.
+        """
+        return [
+            {"type": "prefix", "route": "/admin", "method": "GET"},
```

Comment:
can we support something like `"method": "*"` which bans all methods in admin route. I think its needed since POST and PUT may contain passwords too